ghedesigner/radial_numerical_borehole.py

```diff
@@ -24,26 +27,8 @@ class RadialNumericalBH(object):
     Energy Storage-EcoStock. Pomona, NJ, May 31-June 2.
     """
 
-    def __init__(
-            self,
-            single_u_tube: SingleUTube,
-            ground_init_temp: float = 20.0,
-            data_type: np.dtype = np.double,
-    ):
```

Comment:
`ground_init_temp` and `data_type` were never used, and I don't expect that to change, so they are now gone.